bin/gwdetchar-omega

```diff
@@ -27,7 +27,10 @@ import re
 import sys
 import ast
 import warnings
-from StringIO import StringIO
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
@scottcoughlin2014 (@alurban) can we swap the ordering of this to make it python3-first?

Reply:
Done